apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/hypr.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id": "hypr-idp",
+    "name": "HYPR",
+    "description": "Enable login for users with existing HYPR accounts.",
+    "image": "hypr",
+    "category": "DEFAULT",
+    "displayOrder": 4,
+    "services": [],
+    "tags": [ "Social-Login" ],
+    "idp": {
+        "name": "HYPR",
+        "description": "",
+        "image": "",
+        "isPrimary": false,
+        "isFederationHub": false,
+        "homeRealmIdentifier": "",
+        "certificate": {
+            "certificates": []
+        },
+        "alias": "https://localhost:9444/oauth2/token",
```

Comment:
Is this expected?

Reply:
yes. alias is not used by the hypr idp. adding a default value similar to other idps